backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -47,8 +46,8 @@ object RuntimeAttributesValidation {
       case Some(WdlInteger(i)) => ContinueOnReturnCodeSet(Set(i)).successNel
       case Some(WdlArray(wdlType, seq)) =>
         val nels: Seq[ErrorOr[Int]] = seq map validateInt
-        val nrFailures: Int = nels count { case p => p.isInstanceOf[Failure[NonEmptyList[String]]] }
-        if (nrFailures == 0) {
+        val numFailures: Int = nels.count(_.isFailure)
+        if (numFailures == 0) {
           val defaultReturnCodeNel = Set.empty[Int].successNel[String]
           nels.foldLeft(defaultReturnCodeNel)((acc, v) => (acc |@| v) { (a, v) => a + v }) map ContinueOnReturnCodeSet
```

Comment:
Only touched this method in the `object RuntimeAttributesValidation` because of my _"must. get. green. intellij. checkmark."_ Otherwise, the functions were generally left behind for the other backends to use as is, while the SFS is now using these methods encapsulated in subclasses of the `trait RuntimeAttributesValidation`.